### PR TITLE
Notes on renamed pickers

### PR DESCRIPTION
**Pull request by sofietoft**
Add notes to the renamed Content Picker and Document Picker to avoid confusion.

### CODE REVIEW

**Review by leekelleher — APPROVED** [review]
@sofietoft  Looks good to me. 👍 Thank you! 🙏 

A note on how we're using the word "Content" in the backoffice UI/code, it's as an umbrella term for either Document, Media or Members.

Of course, we still have the "Content" section (for managing Documents), which will still confuse matters. 😕  I'm sure these will be straightened out in due course.

**Comment by sofietoft** [comment]
Good point @leekelleher !

Does it make sense to add this to note as well?
I've done it already, but can be taken out again if it doesn't make sense.

**Comment by leekelleher** [comment]
@sofietoft Makes sense. 👍👍

File: 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/content-picker.md [hunk 1]
```diff
@@ -4,11 +4,21 @@
 
 `Returns: IEnumerable<IPublishedContent>`
 
-## Settings
+The Content Picker allows you to configure the type of content tree to render and what part of it should be rendered. For content it allows you to select a dynamic root node based on the current document using the Content Picker.
```
[the remark this answers is not in the document]
Comment (eshanrnh) [reply 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/content-picker.md:7]: The first statement is slightly confusing so tried to suggest one.

File: 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/content-picker.md [hunk 2]
```diff
@@ -4,11 +4,21 @@
...
 
-The Content Picker allows you to configure the type of tree to render and what part of the tree should be rendered. For content it allows you to select a dynamic root node based on the current document using the content picker.
+{% hint style="info" %}
+The Content Picker is what was known as the **Multinode Treepicker** in version 13 and below.
```
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/content-picker.md:10]:
```suggestion
The Content Picker was formerly known as the **Multinode Treepicker** in version 13 and below.
```

File: 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/document-picker.md [hunk 3]
```diff
@@ -4,7 +4,15 @@
 
 `Returns: IPublishedContent`
 
-The document picker opens a panel to pick a specific page from the content structure. The value saved is the selected nodes [UDI](../../../../reference/querying/udi-identifiers.md)
+The Document Picker opens a panel to pick a specific page from the content structure. The value saved is the selected nodes [UDI](../../../../reference/querying/udi-identifiers.md).
+
+{% hint style="info" %}
+The Document Picker is was what known as the **Content Picker** in version 13 and below.
```
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/document-picker.md:10]:
```suggestion
The Document Picker was formerly known as the **Content Picker** in version 13 and below.
```

**Review by eshanrnh — APPROVED** [review]
The PR looks great, @sofietoft 🙌 Just a few quick suggestions for you to check out😊

**Comment by sofietoft** [comment]
Thanks for the review @sofietoft ! 💪 
All suggestions makes sense.

**Review by eshanrnh — APPROVED** [review]
Looks good to me 🙌

**Pull request merged**